hw/ip/clkmgr/doc/theory_of_operation.md

```diff
@@ -182,6 +188,9 @@ In cases where there is a frequency mismatch, the clock manager supports clock d
 The divided frequency is not assumed to be synchronous with its source and are thus treated like another asynchronous branch.
```

Comment:
Huh, this section is misleading: I think line 186 should explain clock division is required for peripherals that need clocks slower than IO. In normal operation the IO clock is divided by 2 and 4.

Reply:
I agree - I have reworded this part, PTAL.